src/Symfony/Bundle/FrameworkBundle/Console/Application.php

```diff
@@ -212,15 +212,17 @@ protected function registerCommands()
      */
     public static function bootstrapEnv(array &$argv)
     {
-        for ($i = 0; $i < \count($argv) && '--' !== $v = $argv[$i]; ++$i) {
+        $argvTotal = \count($argv);
```

Comment:
This change is not needed: count is already optimized and a temp variable provides nothing.

Reply:
Thanks for the tip. Done.